Foxtrot/Tests/CRASanitizeTest.cs

```diff
@@ -16,55 +16,44 @@
 using System.Text;
 using System.Collections.Generic;
 using System.Linq;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.IO;
 using System.Xml.Linq;
+using Xunit;
+using Xunit.Abstractions;
 
 namespace Tests
 {
-  [TestClass]
   public partial class Test
   {
-    private TestContext testContextInstance;
+    private readonly ITestOutputHelper _testOutputHelper;
```

Comment:
The idea of abstracting output away looks very promising. But can we think a bit about name of this abstraction?

Maybe something like `ITestDiagnosticStream`, `IOutputStream`, `ITestConsole`?